client/src/sass/_general.scss

```diff
@@ -64,7 +64,7 @@ small {
 
 .hero-page-img {
   height: 320px;
-
+  background-repeat: no-repeat;
```

Comment:
better yet (i think): `background-size:cover;`
it stretches the image a bit, but I don't think it's noticeable.